include/ws_functions.inc.php

```diff
@@ -202,6 +202,10 @@ function ws_flickr2piwigo_importPhoto($params)
       if ($photo['visibility']['isfamily']) $updates['level'] = 4;
       if ($photo['visibility']['isfriend']) $updates['level'] = 2;
     }
+    if (in_array('fill_views', $photo['fills']) && isset($photo['views'])) {
```

Comment:
Is it ever possible that `$photo['views']` can be set to something other than an integer? It might be worth checking here, and/or casting to an int.